stackable-webhook/src/servers/conversion.rs

```diff
@@ -1,7 +1,7 @@
 use std::fmt::Debug;
 
 use axum::{extract::State, routing::post, Json, Router};
-use kube::core::conversion::ConversionReview;
+pub use kube::core::conversion::ConversionReview;
```

Comment:
I'm honestly not sure, but I **think** in the past we tried to get rid of re-exports in operator-rs (something something about picking versions and stuff), but @nightkr knows best.

Reply:
I would be happy to leave it as is, but we then require an explicit dependency. This is mostly relevant for the guide in https://github.com/stackabletech/documentation/pull/561.

The current example needs adjustment and we should also add a hint for the developer to keep the dependency version in sync between `stackable-webhook` and the operator.

Reply:
One small follow-up: I would keep the change in this line https://github.com/stackabletech/operator-rs/blob/d34e77bafbea6d2565235448910339678b83d0d7/stackable-webhook/src/lib.rs#L38

It makes a little more sense to import `ConversionWebhookServer` (and other servers in the future) from the `servers` module instead of the root level.

Reply:
> I'm honestly not sure, but I think in the past we tried to get rid of re-exports in operator-rs (something something about picking versions and stuff), but @nightkr knows best.

Largely, the other way around. The operators _need_ to use the same version of kube-rs as op-rs, so it's preferable if we just have to decide the version in op-rs and then re-export from there.

Reply:
Adjusted in 5314e8e

Reply:
I don't think that comment adds any value IMO, it's _already_ selected by virtue of having the `pub`. If you do think it warrants a comment, mention ___WHY___ it warrants a re-export.

Reply:
Clarified in 46dfae2